packages/terra-form-select/src/shared/_MenuUtil.js

```diff
@@ -331,6 +331,8 @@ class MenuUtil {
     });
   }
 
+  static isMac () { return navigator.userAgent.indexOf('Mac') !== -1 && navigator.userAgent.indexOf('Win') === -1 };
```

Comment:
Is the second condition needed? I would assume that if the OS is Mac then it's automatically not windows.